File: data/gpt-4o/readable_format_HPEC/cpp/file218.cpp
#include <iostream>
#define DATA_PRINTF_MODIFIER "%d "

void print_array(int n, int path[][100])
{
 for(int i=0; i<n; i++)
 {
  for(int j=0; j<n; j++)
  {
   printf(DATA_PRINTF_MODIFIER, path[j][i]);
   if (((i * n) + j) % 20 == 0) 
   {
    printf("
");
   }
  }
 }
 printf("
");
}

File: data/CodeLlama/readable_format_HPEC/fortran/file218.f90
subroutine print_array(n, path)
implicit none

DATA_TYPE, dimension(n, n) :: path
integer :: i, j, n

do i=1, n
do j=1, n
write(0, DATA_PRINTF_MODIFIER) path(j,i)

if (mod(((i - 1) * n) + j - 1, 20) == 0) then
write(0, *)
end if
end do
end do
write(0, *)
end subroutine
